CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# Unreleased
+
```

Comment:
Updating the changelog is an opportunity to respect [the rules](https://github.com/openzim/_python-bootstrap/wiki/CHANGELOG.md). Add the header and fix the release formatting and for this new release, use the proper sub-sections

Reply:
This is a very good point. I have opened an issue #179 to perform all changes needed to respect the rules. And this issue will have to be treated before releasing. This would avoid to have more confusion in this PR than there is already (there is already too many topics in one PR). I'm tired to have code which does not match Flake8 / black rules for instance.